src/types.ts

```diff
@@ -211,3 +211,20 @@ export type AppClient = {
    */
   requirePkce: boolean;
 };
+
+export type UserInfo = {
+  name: string;
+  middleName?: string;
+  givenName?: string;
+  phoneNumber?: string;
+  phoneNumberVerified?: boolean;
+  email?: string;
+  emailVerified?: boolean;
+  familyName?: string;
+  birthDate?: string;
+  address?: string;
+  locale?: string;
+  createdAt: Date;
+  modifiedAt: Date;
+  zoneInfo?: string;
```

Comment:
I think none of these should allow undefined, but all of them should allow `null`. This is more consistent with other parts in the codebase.

Reply:
Oh yeahh `null`, a database thing!~